### DISCUSSION TITLE
Grid consumption sensor selection

### DISCUSSION THREAD

**Discussion by CriticalO** [BottlecapDave/HomeAssistant-OctopusEnergy#1134] [Q&A] [answered]
Hello and thanks for such a great integration for HA, we ordered smart metres and a home mini just so we could use it in home assistant. Anyway all installed great except we had two electricity metres which showed an repair error in HA, although we figured out in the end that the second one was the old metre, so I just disabled it, deleted it and ignored the repair warning.

The bit I'm confused about it which sensor to choose for grid consumption for both electricity and gas, there's loads of them in there and I ended up choosing... Current Accumulative Consumption
Electricity for the electric as there was a current accumulative cost which matched the value and I thought I could use this for the x value of the grid. Then I used the same set up for the gas, so Current Accumulative Consumption (m3) Gas, and current accumulative cost. Have I chosen the correct entries for grid consumption for both gas and electricity?

Thanks I'll attach a screen shoot of what it looks like with the current sensors...

![image](https://github.com/user-attachments/assets/50ccb06c-fd90-476b-8de9-be6c122b61b8)

Anyway thanks hopefully someone can advise me on if I've got the right sensors of if I need to choose different sensors? Much appreciated.

Shane

**Comment by trevorwarwick** [comment, marked as answer]
If you look at the documentation, it provides different options depending on your setup

https://bottlecapdave.github.io/HomeAssistant-OctopusEnergy/setup/energy_dashboard/

**Reply by CriticalO** [reply]
Awesome thanks very much that's exactly what I needed much appreciated. 

Have a good one and thanks again for a really useful integration :D
